src/main/java/Task.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * represents a task
+ */
+public class Task {
+    private String type;
+    private String todo;
+    private LocalDate date;
+    private boolean isDone;
+
+    Task(String type, String todo, LocalDate date, boolean isDone) {
+        this.type = type;
+        this.todo = todo;
+        this.date = date;
+        this.isDone = isDone;
+    }
+
+    public String getTodo() {
+        return todo;
+    }
+
+    public void setStatus(boolean newStatus) {
```

Comment:
Perhaps the newStatus could be better named as a boolean variable like isDoneNow.